cupy_backends/cuda/api/_runtime_typedef.pxi

```diff
@@ -93,7 +93,7 @@ cdef extern from *:
         MemoryKind kind
 
     ctypedef struct TextureDesc 'cudaTextureDesc':
-        int addressMode[3]
+        TextureAddressMode addressMode[3]
```

Comment:
It is puzzling that you ran into issues 🤔 `TextureAddressMode` is just a typedef of `int` (line 30) so this change alone should make no difference as far as Cython is concerned?